client/landing/gutenboarding/constants.ts

```diff
@@ -7,3 +7,10 @@
  * @see https://stackoverflow.com/a/44755058/1432801
  */
 export const selectorDebounce = 300;
+
+/**
+ * Flag for temporarily hiding the sidebar and related toggle button.
+ *
+ * @todo Remove once we re-enact the sidebar or decide to go without (or via another route).
+ */
+export const enableSidebarDisplay = false
```

Comment:
Aside, this is more of a feature flag, rather than an app constant. I'd advise against conflating the two concepts.

For feature flags, we actually have [precedent](https://github.com/Automattic/wp-calypso/blob/4bc964190c1305dd448df58cb75ae63623ba7af1/client/landing/gutenboarding/index.tsx#L22) of using Calypso's `config` system in Gutenboarding. This comes with its own set of problems, but for now, we can accept it as an established pattern; if we decide to change it, we can apply whatever change to all instances.

Reply:
Good aside. `config` was the first place I looked at, but I could hear the argument

> tight coupling to other parts of Calypso, which we generally try to avoid in Gutenboarding

coming. Never know with you guys :p